package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "resin-procbots",
+  "version": "0.0.1",
+  "description": "Process robots for automating CI pipelines",
+  "homepage": "https://github.com/resin-io-modules/resin-procbots#readme",
+  "preferGlobal": true,
+  "types": "lib/**/*.ts",
```

Comment:
I think (though I'm not 100% sure) `types` is supposed to point to a compiled set of type definitions, rather than the an entire typed implementation. That's certainly far more common. See [resin-sdk-preconfigured](https://github.com/resin-io-modules/resin-sdk-preconfigured) for example, which outputs `.js`, `.js.map` and `.d.ts` (just set `declaration: true` in tsconfig) into `/build`